ckanext/dcat/harvesters/base.py

```diff
@@ -159,17 +159,12 @@ def _read_datasets_from_db(self, guid):
         '''
         Returns a database result of datasets matching the given guid.
         '''
-        if toolkit.check_ckan_version(max_version="2.11"):
```

Comment:
This was probably changed by mistake

Reply:
Unfortunately, when I revert this setting, the pipeline fails. Please refer to the following link for details: https://github.com/ckan/ckanext-dcat/actions/runs/12284575222. The tests seem to fail unexpectedly.

Reply:
Maybe something upstream changed?

Reply:
My bad, the version check was wrong: https://github.com/ckan/ckanext-dcat/commit/6bcf6931158ce995487a0828accf926ecfb375b1